### ISSUE TITLE
Container exits after first backup

### ISSUE THREAD

**Issue by AaronWiora** [itzg/docker-mc-backup#48]
After the first backup no further backups followed so I tried to understand what's happening. The container keeps stopping after the first backup.

**Compose File**

```
version: '3.7'
services:
  mc:
    image: itzg/minecraft-server
    ports:
    - 25565:25565
    environment:
      EULA: "TRUE"
      OVERRIDE_SERVER_PROPERTIES: "TRUE"
      MOTD: "..."
      ICON: "..."
      VIEW_DISTANCE: "16"
      SERVER_NAME: "..."
      MEMORY: 3G
    volumes:
    - mc:/data
  backups:
    image: itzg/mc-backup
    environment:
      BACKUP_INTERVAL: "1h"
      # instead of network_mode below, could declare RCON_HOST
      # RCON_HOST: mc
    volumes:
    # mount the same volume used by server, but read-only
    - mc:/data:ro
    # use a host attached directory so that it in turn can be backed up
    # to external/cloud storage
    - ./mc-backups:/backups
    # share network namespace with server to simplify rcon access
    network_mode: "service:mc"

volumes:
  mc: {}

```

**Container Overview**
```

CONTAINER ID   IMAGE                             COMMAND                  CREATED        STATUS                    PORTS                                                                                  NAMES
048040e452d6   itzg/mc-backup                    "/opt/entrypoint-dem…"   46 hours ago   Exited (1) 45 hours ago                                                                                          minecraft_backups_1
136670636adf   itzg/minecraft-server             "/start"                 46 hours ago   Up 46 hours (healthy)     0.0.0.0:25565->25565/tcp, :::25565->25565/tcp, 25575/tcp                               minecraft_mc_1
```

**Log**
```
2021-12-05T17:57:15+0000 INFO waiting initial delay of 2m...
2021-12-05T17:59:15+0000 INFO waiting for rcon readiness...
2021-12-05T17:59:15+0000 INFO Command executed successfully rcon-cli save-on
2021-12-05T17:59:15+0000 INFO Command executed successfully rcon-cli save-off
2021-12-05T17:59:15+0000 INFO Command executed successfully rcon-cli save-all
2021-12-05T17:59:15+0000 INFO Command executed successfully sync
2021-12-05T17:59:15+0000 INFO Backing up content in /data to /backups/world-20211205-175915.tgz
2021-12-05T17:59:20+0000 INFO Command executed successfully rcon-cli save-on
2021-12-05T17:59:20+0000 INFO sleeping 1h...
2021-12-05T18:59:20+0000 INFO Command executed successfully rcon-cli save-off
2021-12-05T18:59:20+0000 INFO Command executed successfully rcon-cli save-all
2021-12-05T18:59:20+0000 INFO Command executed successfully sync
2021-12-05T18:59:20+0000 INFO Backing up content in /data to /backups/world-20211205-185920.tgz
tar: ./world/entities/r.2.1.mca: file changed as we read it
tar: ./world/entities/r.0.0.mca: file changed as we read it
tar: ./world/entities/r.1.0.mca: file changed as we read it
2021-12-05T18:59:27+0000 INFO Command executed successfully rcon-cli save-on
```

**Inspect**
               
```
[
    {
        "Id": "048040e452d6f4618ab606966db896b2c551581d824929a3cd547fc11438451e",
        "Created": "2021-12-05T17:57:15.140760585Z",
        "Path": "/opt/entrypoint-demoter",
        "Args": [
            "--match",
            "/backups",
            "backup",
            "loop"
        ],
        "State": {
            "Status": "exited",
            "Running": false,
            "Paused": false,
            "Restarting": false,
            "OOMKilled": false,
            "Dead": false,
            "Pid": 0,
            "ExitCode": 1,
            "Error": "",
            "StartedAt": "2021-12-05T17:57:15.326928937Z",
            "FinishedAt": "2021-12-05T18:59:27.375662406Z"
        },
        "Image": "sha256:cb8691c1bf155a096791c02918c07927fc377c932934fb69ba69ee4f8e2e6ec5",
        "ResolvConfPath": "/var/lib/docker/containers/136670636adf73b097df51ef08fd43ad596789b35f443dc31b0296da96dcae5f/resolv.conf",
        "HostnamePath": "/var/lib/docker/containers/136670636adf73b097df51ef08fd43ad596789b35f443dc31b0296da96dcae5f/hostname",
        "HostsPath": "/var/lib/docker/containers/136670636adf73b097df51ef08fd43ad596789b35f443dc31b0296da96dcae5f/hosts",
        "LogPath": "/var/lib/docker/containers/048040e452d6f4618ab606966db896b2c551581d824929a3cd547fc11438451e/048040e452d6f4618ab606966db896b2c551581d824929a3cd547fc11438451e-json.log",
        "Name": "/minecraft_backups_1",
        "RestartCount": 0,
        "Driver": "overlay2",
        "Platform": "linux",
        "MountLabel": "",
        "ProcessLabel": "",
        "AppArmorProfile": "docker-default",
        "ExecIDs": null,
        "HostConfig": {
            "Binds": [
                "minecraft_mc:/data:ro",
                "/home/games/minecraft/mc-backups:/backups:rw"
            ],
            "ContainerIDFile": "",
            "LogConfig": {
                "Type": "json-file",
                "Config": {}
            },
            "NetworkMode": "container:136670636adf73b097df51ef08fd43ad596789b35f443dc31b0296da96dcae5f",
            "PortBindings": {},
            "RestartPolicy": {
                "Name": "",
                "MaximumRetryCount": 0
            },
            "AutoRemove": false,
            "VolumeDriver": "",
            "VolumesFrom": [],
            "CapAdd": null,
            "CapDrop": null,
            "CgroupnsMode": "private",
            "Dns": null,
            "DnsOptions": null,
            "DnsSearch": null,
            "ExtraHosts": null,
            "GroupAdd": null,
            "IpcMode": "private",
            "Cgroup": "",
            "Links": null,
            "OomScoreAdj": 0,
            "PidMode": "",
            "Privileged": false,
            "PublishAllPorts": false,
            "ReadonlyRootfs": false,
            "SecurityOpt": null,
            "UTSMode": "",
            "UsernsMode": "",
            "ShmSize": 67108864,
            "Runtime": "runc",
            "ConsoleSize": [
                0,
                0
            ],
            "Isolation": "",
            "CpuShares": 0,
            "Memory": 0,
            "NanoCpus": 0,
            "CgroupParent": "",
            "BlkioWeight": 0,
            "BlkioWeightDevice": null,
            "BlkioDeviceReadBps": null,
            "BlkioDeviceWriteBps": null,
            "BlkioDeviceReadIOps": null,
            "BlkioDeviceWriteIOps": null,
            "CpuPeriod": 0,
            "CpuQuota": 0,
            "CpuRealtimePeriod": 0,
            "CpuRealtimeRuntime": 0,
            "CpusetCpus": "",
            "CpusetMems": "",
            "Devices": null,
            "DeviceCgroupRules": null,
            "DeviceRequests": null,
            "KernelMemory": 0,
            "KernelMemoryTCP": 0,
            "MemoryReservation": 0,
            "MemorySwap": 0,
            "MemorySwappiness": null,
            "OomKillDisable": null,
            "PidsLimit": null,
            "Ulimits": null,
            "CpuCount": 0,
            "CpuPercent": 0,
            "IOMaximumIOps": 0,
            "IOMaximumBandwidth": 0,
            "MaskedPaths": [
                "/proc/asound",
                "/proc/acpi",
                "/proc/kcore",
                "/proc/keys",
                "/proc/latency_stats",
                "/proc/timer_list",
                "/proc/timer_stats",
                "/proc/sched_debug",
                "/proc/scsi",
                "/sys/firmware"
            ],
            "ReadonlyPaths": [
                "/proc/bus",
                "/proc/fs",
                "/proc/irq",
                "/proc/sys",
                "/proc/sysrq-trigger"
            ]
        },
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/22766048d958e6c38211f56615c7a4645cc9c43dd75a7e28ca997c93138d056c-init/diff:/var/lib/docker/overlay2/02a1bdc61026ccc8e365d33791f4c040ccb91e8541a22b875b7da1dee28f4484/diff:/var/lib/docker/overlay2/928abf623227044d80f253ccd044c2d7b6667940e93b5f9210d8c1f2d450c1e2/diff:/var/lib/docker/overlay2/99e1777f2bb825401269c36e3d9b0b04aa9890990bece5b1677b60785f25040b/diff:/var/lib/docker/overlay2/6ec8afb3476e10c65028a60261e88feb24f8af368610a87406701f6fb91e5716/diff:/var/lib/docker/overlay2/a1c0b6b5656cfa8d6730cf42496481866f88348438ccf50eda33139ed84976c1/diff:/var/lib/docker/overlay2/35ed11231ce0397b6952b1aef9b6062952de0ec996419d194e93c7530fb3b7fd/diff:/var/lib/docker/overlay2/bbebe113994837afe342f9ff4fbfaf65965294d4f16d4ea12ed671c681f8c138/diff:/var/lib/docker/overlay2/82415d761cbc91dc772ff4c254c3108dfb376d303695cb5301f9b53608377d28/diff:/var/lib/docker/overlay2/38835db10885bb933b29311e124a6bce870209ca8266c3ff187c1f0519a44d84/diff:/var/lib/docker/overlay2/a1c0fdfc2fa2d33f459ef30603985ae182b434dadf9316b731998a93f9670508/diff:/var/lib/docker/overlay2/7ebca3febc5f3cc95b22866656cc587f27e480b2e59f1abf5aae34e8077056c7/diff:/var/lib/docker/overlay2/bd54835b3f2351256b9b0a572633d4730ef6d32d2983ea560d69753c0208d804/diff:/var/lib/docker/overlay2/b589e1a8c81d492a195a58f36980344ba15c55ec7f7f6cf11035431bbd470896/diff:/var/lib/docker/overlay2/a732244fc0a63377e19880c3ceabcbb9ef912558ce94667a32e56efb3564748b/diff",
                "MergedDir": "/var/lib/docker/overlay2/22766048d958e6c38211f56615c7a4645cc9c43dd75a7e28ca997c93138d056c/merged",
                "UpperDir": "/var/lib/docker/overlay2/22766048d958e6c38211f56615c7a4645cc9c43dd75a7e28ca997c93138d056c/diff",
                "WorkDir": "/var/lib/docker/overlay2/22766048d958e6c38211f56615c7a4645cc9c43dd75a7e28ca997c93138d056c/work"
            },
            "Name": "overlay2"
        },
        "Mounts": [
            {
                "Type": "volume",
                "Name": "minecraft_mc",
                "Source": "/var/lib/docker/volumes/minecraft_mc/_data",
                "Destination": "/data",
                "Driver": "local",
                "Mode": "ro",
                "RW": false,
                "Propagation": ""
            },
            {
                "Type": "bind",
                "Source": "/home/games/minecraft/mc-backups",
                "Destination": "/backups",
                "Mode": "rw",
                "RW": true,
                "Propagation": "rprivate"
            }
        ],
        "Config": {
            "Hostname": "136670636adf",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "BACKUP_INTERVAL=1h",
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "HOME=/tmp"
            ],
            "Cmd": [
                "backup",
                "loop"
            ],
            "Image": "itzg/mc-backup",
            "Volumes": {
                "/backups": {},
                "/data": {}
            },
            "WorkingDir": "",
            "Entrypoint": [
                "/opt/entrypoint-demoter",
                "--match",
                "/backups"
            ],
            "OnBuild": null,
            "Labels": {
                "com.docker.compose.config-hash": "78c3e039b5dfc3414c1c5443870fbe1124fcc5d1d266369349e167464e40fbcb",
                "com.docker.compose.container-number": "1",
                "com.docker.compose.oneoff": "False",
                "com.docker.compose.project": "minecraft",
                "com.docker.compose.project.config_files": "docker-compose.yml",
                "com.docker.compose.project.working_dir": "/home/games/minecraft",
                "com.docker.compose.service": "backups",
                "com.docker.compose.version": "1.29.2"
            }
        },
        "NetworkSettings": {
            "Bridge": "",
            "SandboxID": "",
            "HairpinMode": false,
            "LinkLocalIPv6Address": "",
            "LinkLocalIPv6PrefixLen": 0,
            "Ports": {},
            "SandboxKey": "",
            "SecondaryIPAddresses": null,
            "SecondaryIPv6Addresses": null,
            "EndpointID": "",
            "Gateway": "",
            "GlobalIPv6Address": "",
            "GlobalIPv6PrefixLen": 0,
            "IPAddress": "",
            "IPPrefixLen": 0,
            "IPv6Gateway": "",
            "MacAddress": "",
            "Networks": {}
        }
    }
]
```

**Comment by StoneLabs** [comment]
This is most likely fixed #47. 
Try updating your container, and see if it works now.

Also somewhat related to #49.

**Comment by AaronWiora** [comment]
> 

Thanks for the heads up I did not update my container for 1 week. 
I will close this issue as soon as I have verfied that it's working.
